### ISSUE TITLE
[Bug]: Unable to reproduce the results of supersimplenet on the MVTec dataset.

### ISSUE THREAD

**Issue by TripleJoy** [open-edge-platform/anomalib#2688]
### Describe the bug

I’m trying to reproduce the reported results on the MVTec dataset using the `supersimplenet` model, but unfortunately, I haven’t been able to achieve similar performance. During training, it appears that the loss does not converge, even after several epochs.

Could you please share more details about the training or evaluation setup? 

### Dataset

MVTecAD

### Model

Other (please specify in the field below)

### Steps to reproduce the behavior

none

### OS information

OS information:
- OS: [Ubuntu 20.04]
- Python version: [3.10.0]
- Anomalib version: [2.1.0.dev0]
- PyTorch version: [2.5.0]
- CUDA/cuDNN version: [12.4]
- GPU models and configuration: [Tesla V100]

### Expected behavior

none

### Pip/GitHub

GitHub

### What version/branch did you use?

2.1.0.dev0

### Configuration YAML

```yaml
none
```

### Logs

```shell
none
```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by samet-akcay** [comment]
@blaz-r any thoughts?

**Comment by blaz-r** [comment]
Quite unfortunate. Probably something from v2 to v2.1 broke the method. I'll check what's going on. My suspicion is on the feature extractor. I'm really busy right now so I'll get back to you after the 15th, but if you have the capacity you could check the feature extractor if weights are properly loaded.

**Comment by TripleJoy** [comment]
Sure, good luck!

**Comment by samet-akcay** [comment]
@TripleJoy, which anomalib version are you using?

v2.1 has not been released officially. If it is versioning related, pypi version should ideally work fine

**Comment by TripleJoy** [comment]
I am using version 2.1.0.dev0. I will try other versions to see if they work properly.

**Comment by blaz-r** [comment]
The issue was indeed in the feature extractor. I fixed the pretrained weights name in #2712 and I think it should now work. I'll still have to verify on all categories.

**Comment by TripleJoy** [comment]
Okay, thank you for your work.
